docs/src/main/asciidoc/amazon-dynamodb.adoc

```diff
@@ -321,7 +321,7 @@ The implementation is pretty straightforward and you just need to define your en
 == Configuring DynamoDB clients
 
 Both DynamoDB clients (sync and async) are configurable via the `application.properties` file that can be provided in the `src/main/resources` directory.
-Additionally, you need to added to the classpath a proper implementation of the sync client. By default the extension uses URL connection HTTP client, so
+Additionally, you need to add to the classpath a proper implementation of the sync client. By default the extension uses the URL connection HTTP client, so
```

Comment:
> By default the extension uses the URL connection HTTP client

Hello @gunnarmorling, while we are at it, perhaps we should change this to "By default the extension uses the `java.net.URLConnection` HTTP client..."

Reply:
Done.